Source/ablastr/utils/SignalHandling.cpp

```diff
@@ -162,14 +178,19 @@ SignalHandling::CheckSignals()
 
 #if defined(AMREX_USE_MPI)
     auto comm = amrex::ParallelDescriptor::Communicator();
+    static_assert(sizeof(bool) == 1, "We communicate bools as 1 byte-sized type in MPI");
```

Comment:
Please comment that this is an accommodation for Cray's deficient MPI implementation. They support the MPI 3 asynchronous collective, but not the simple datatype definition.

Reply:
Yeah, it's a pitty!
Will add the same description as in the PR to make clear why we hack back to bytes